examples/Demo/FluentUI.Demo.Client/Documentation/Components/Button/Examples/ButtonIcon.razor

```diff
@@ -13,3 +13,13 @@
     <FluentIcon Value="@(new Icons.Regular.Size20.Globe())" Color="Color.Error" Slot="start" />
     Button
 </FluentButton>
+
+
+<FluentButton IconOnly="true">
+    <svg viewBox="0 0 20 20">
```

Comment:
Can you use the `FluentIcon` component and not HTML sample? We need to promote the lib 😀

Reply:
I want to emphasize the case of using a custom icon. If it was a FluentIcon, I could just as well use IconStart/IconEnd